### PR TITLE
Dorothea Business Site

### PR DESCRIPTION
**Pull request by Dorothea-B**
https://guitar-site-business.netlify.app/

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -4,27 +4,140 @@
...
+        <h2>Become instantly cooler with our intensive training program.</h2>
+      </div>
+
+      <video autoplay loop muted preload="auto" poster="./images/pexels-edward-eyer-811838.jpg">
+        <source src="./videos/guitar-vid.mp4" type="video/mp4" alt="guitar playing">
+        <!-- <source src="./big_buck_bunny.webm" type="video/webm"> -->
+        <p>This browser doesn't support video...</p>
+      </video>
+     
+    </section>
```
Comment (Mimmisen) [inline code/index.html:34]: easy to read and follow along!

File: code/index.html [hunk 2]
```diff
@@ -4,27 +4,140 @@
...
+              
+                
+              <label for="fname">Name:</label><br>
+                <input type="text" id="fname" name="firstname" required><br>
+              <label for="email">Email:</label><br>
+                <input type="email" id="email" name="emailname" required><br>
+              <label for="nmbr">Phone:</label><br>
+                <input type="number" id="nmbr" name="phonenumber"><br> 
+              <label for="message">Custom message<br> </label><br>
+                  <textarea name="message" rows="5" cols="50%">Special requests etc</textarea><br>
```
Comment (Mimmisen) [inline code/index.html:102]: what does the <br> do in the form? I can’t see in the inspector what their function is there

File: code/style.css [hunk 3]
```diff
@@ -1,7 +1,490 @@
...
+body {
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+} 
+
+section {
+
+  flex: 1;
+  width: 100%
```
Comment (Mimmisen) [inline code/style.css:19]: you are missing a ;

File: code/style.css [hunk 4]
```diff
@@ -1,7 +1,490 @@
...
+  font-weight: 500;
+  font-size: 1rem;
+}
+
+button {
+  background: none;
+	color: inherit;
+	border: none;
+	padding: 0;
+	font: inherit;
```
Comment (Mimmisen) [inline code/style.css:145]: I can see that most of the css is written with a space between your attribute/value and its declarations but there are some places where it looks different, maybe you could take a quick check that everything looks cohesive before you are done

File: code/style.css [hunk 5]
```diff
@@ -1,7 +1,490 @@
...
+  flex: 1;
+  max-width: 400px;
+  min-width: 310px;
+  
+}
+
+.left {
+  flex: 1;
+  max-width: 400px;
+
```
Comment (Mimmisen) [inline code/style.css:245]: maybe you could give right and left names that make you understand where they are in the html

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,490 @@
...
+      video {
+
+        height: 100%;
+
+      }
+
+      h1 {
+        font-size: 2rem;
+        margin-top: 5rem;
+        padding-right: 2rem;;
```
Comment (Mimmisen) [inline code/style.css:341]: here you have 2 ; on 341 and on line 453

**Review by Mimmisen — APPROVED** [review]
You have done all the requirements! You have made a beautiful page, I really like that you have an image behind the sign up form, looks professional. Your html was easy to read and follow, my feedback I’ve already written in comments but overall a great job!

File: code/index.html (hunk 2 above)
Comment (Dorothea-B) [reply code/index.html:102]: I used br to break the line between the label texts and the form boxes!

File: code/style.css (hunk 5 above)
Comment (Dorothea-B) [reply code/style.css:245]: Yes, good idea.

File: code/style.css (hunk 6 above)
Comment (Dorothea-B) [reply code/style.css:341]: Thank you for your observant and thorough code review! 🌟

**Pull request closed without merging**